analyzer/src/funTest/kotlin/AnalyzerFunTest.kt

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
commit-msg: To me it is not clear why exactly the clean-up has to happen only after all package managers have finished the resolution. Can you help me understand?

Reply:
For React-Native setups, `npm install` creates Gradle projects on-the-fly inside the `node_modules` directory. If the latter directory is removed before Gradle analysis runs, then the analysis will fail as required projects / definition files are not found.